test/data/user.ts

```diff
@@ -2,6 +2,12 @@ import { REALM_MEMBER_PERMISSIONS, RealmPermissions } from "types/permissions";
 
 import { UserData } from "../../types/rest/user";
 
+export const BOBATAN_ID = "c6HimTlg2RhVH3fC1psXZORdLcx2";
+export const JERSEY_DEVIL_ID = "fb2";
+export const ONCEST_ID = "fb3";
+export const SEXY_ID = "fb4";
+export const ZODIAC_KILLER_ID = "fb5";
+
```

Comment:
only just noticed that user ids are defined in `test/data/auth.ts`, gonna be removing the ones i added here LOL, easy to move over if we feel it's better suited for here